### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by rotimi-babalola**
- connects to online database for project
- creates dummy database table

### CODE REVIEW

File: src/config/config.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+require('dotenv').config()
+
+module.exports = {
+  development: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+    operatorsAliases: {
+      $and: Op.and,
+      $or: Op.or,
+      $eq: Op.eq,
```
Comment (benfluleck) [inline src/config/config.js]: put these all in an object in another file and spread it. this code does not look good.

File: src/config/config.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      $lt: Op.lt,
+      $lte: Op.lte,
+      $like: Op.like
+    }
+  },
+  test: {
+    username: 'root',
+    password: null,
+    database: 'database_test',
+    host: '127.0.0.1',
```
Comment (benfluleck) [inline src/config/config.js]: please put this in the env file. If you can I would love to use sqlite here as the dialect. As testing can be done in memory.

File: src/config/config.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    dialect: 'postgres'
+  },
+  production: {
+    username: 'root',
```
Comment (benfluleck) [inline src/config/config.js]: Get rid of this as it is not being used

File: src/models/dummyhotel.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+    location: DataTypes.STRING
+  }, {
+    classMethods: {
+      associate: function(models) {
+        // associations can be defined here
+      }
+    }
+  });
+  return DummyHotel;
+};
```
Comment (benfluleck) [inline src/models/dummyhotel.js]: new line needed here

File: src/migrations/20190115132117-create-dummy-hotel.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
+'use strict'
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('DummyHotels', {
+      id: {
```
Comment (benfluleck) [inline src/migrations/20190115132117-create-dummy-hotel.js:5]: Change this to es6 methods here is an article that will help.
[https://codewithhugo.com/using-es6-classes-for-sequelize-4-models/](here)

File: src/models/index.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+var config = require(__dirname + '/../config/config.js')[env]
+var db = {}
+
+if (config.use_env_variable) {
+  var sequelize = new Sequelize(process.env[config.use_env_variable])
+} else {
+  var sequelize = new Sequelize(
+    config.database,
+    config.username,
+    config.password,
```
Comment (benfluleck) [inline src/models/index.js:17]: This bit is not being used

File: src/migrations/20190115132117-create-dummy-hotel.js (hunk 5 above)
Comment (rotimi-babalola) [reply src/migrations/20190115132117-create-dummy-hotel.js:5]: I'm not sure you can convert the migrations to es6. I only converted the models to es6 methods

File: src/models/index.js (hunk 6 above)
Comment (rotimi-babalola) [reply src/models/index.js:17]: It could be used in a different environment e.g. testing

File: src/config/config.js (hunk 2 above)
Comment (benfluleck) [reply src/config/config.js]: @rotimi-babalola  this?

**Pull request merged**